messaginginapp/src/main/java/io/customer/messaginginapp/ui/bridge/EngineWebViewDelegate.kt

```diff
@@ -0,0 +1,23 @@
+package io.customer.messaginginapp.ui.bridge
+
+import android.view.View
+import io.customer.messaginginapp.gist.data.model.engine.EngineWebConfiguration
+import io.customer.messaginginapp.gist.presentation.engine.EngineWebView
+import io.customer.messaginginapp.gist.presentation.engine.EngineWebViewListener
+
+/**
+ * Delegate interface to decouple [EngineWebView] from its consumers.
+ * Provides a minimal set of operations to control and configure the view without directly
+ * depending on its implementation.
+ * Useful for testing, mocking, or swapping the view behind the interface.
+ */
+internal interface EngineWebViewDelegate {
+    var listener: EngineWebViewListener?
+
+    fun setup(configuration: EngineWebConfiguration)
+    fun stopLoading()
+    fun releaseResources()
+    fun getView(): View
+    fun setAlpha(alpha: Float)
+    fun bringToFront()
```

Comment:
I reused same names from `View` class in this delegate to avoid reimplementation and keep the overall implementation simpler.